### ISSUE TITLE
Create Cluster Select Zone

### ISSUE THREAD

**Issue by rosera** [GoogleCloudPlatform/cloud-code-vscode#116]
**Environment**:
*(Hint: "Report Kubernetes Extension Issue on Github" command will fill these out for you.)*
Extension version:0.0.9
VSCode version: 1.35
OS: Debian 9

**Description**:
When creating a GKE Cluster, the zone/region drop downs are not being populated consistently. It looks like the script to derive the zone/region is not being invoked correctly. However if the list doesnt populate, then it is impossible to use the wizard to complete the cluster creation. Ideally it should provide the ability to add the zone/region manually and validate post entry.

**Repro step**:
Runing VS Code application on a Debian 9 virtual machine + VS Code application with Docker, behaviour is displayed on both instances.

![vscode_zone_dropdown](https://user-images.githubusercontent.com/1753857/59262798-d7048b80-8c37-11e9-891b-44234f2f1da2.png)

**Comment by redborian** [comment]
Thanks for reporting the issue @rosera. From your experience, what is the rate of successfully loading zones/regions? Do you see the progress bar below the drop-down run for sometime before stopping? Do you see any errors being shown? Can you run this command reliably? `gcloud compute zones list`?
Also, can you share your gcloud version?

**Comment by rosera** [comment]
I can see an attempt to load the information when the create cluster wizard is initiated. However I did notice that the default project id is never populated on entry to this screen. I am assuming this is linked to the availability of the region/zone cluster list? Is there something I can check that would enable the default project id to be picked up from the wizard?

When I enter a project id - this is accepted, but the zone/region list is still not accessible at this point.

Rate of successful loading - 0%
Progress bar - yes I can see this attempting to load the information
Any errors - no errors displayed


Running gcloud --version 

```
gcpstaging70699_student@vscode-vm:~$ gcloud --version
Google Cloud SDK 250.0.0
alpha 2019.06.07
beta 2019.06.07
bq 2.0.43
core 2019.06.07
gsutil 4.38
kubectl 2019.06.07
```

Running gcloud compute zones list 

`gcpstaging70699_student@vscode-vm:~$ gcloud compute zones list
NAME                       REGION                   STATUS  NEXT_MAINTENANCE  TURNDOWN_DATE
us-east1-b                 us-east1                 UP
us-east1-c                 us-east1                 UP
us-east1-d                 us-east1                 UP
us-east4-c                 us-east4                 UP
us-east4-b                 us-east4                 UP
us-east4-a                 us-east4                 UP
us-central1-c              us-central1              UP
us-central1-a              us-central1              UP
us-central1-f              us-central1              UP
us-central1-b              us-central1              UP
us-west1-b                 us-west1                 UP
us-west1-c                 us-west1                 UP
us-west1-a                 us-west1                 UP
europe-west4-a             europe-west4             UP
europe-west4-b             europe-west4             UP
europe-west4-c             europe-west4             UP
europe-west1-b             europe-west1             UP
europe-west1-d             europe-west1             UP
europe-west1-c             europe-west1             UP
europe-west3-c             europe-west3             UP
europe-west3-a             europe-west3             UP
europe-west3-b             europe-west3             UP
europe-west2-c             europe-west2             UP
europe-west2-b             europe-west2             UP
europe-west2-a             europe-west2             UP
asia-east1-b               asia-east1               UP
asia-east1-a               asia-east1               UP
asia-east1-c               asia-east1               UP
asia-southeast1-b          asia-southeast1          UP
asia-southeast1-a          asia-southeast1          UP
asia-southeast1-c          asia-southeast1          UP
asia-northeast1-b          asia-northeast1          UP
asia-northeast1-c          asia-northeast1          UP
asia-northeast1-a          asia-northeast1          UP
asia-south1-c              asia-south1              UP
asia-south1-b              asia-south1              UP
asia-south1-a              asia-south1              UP
australia-southeast1-b     australia-southeast1     UP
australia-southeast1-c     australia-southeast1     UP
australia-southeast1-a     australia-southeast1     UP
southamerica-east1-b       southamerica-east1       UP
southamerica-east1-c       southamerica-east1       UP
southamerica-east1-a       southamerica-east1       UP
asia-east2-a               asia-east2               UP
asia-east2-b               asia-east2               UP
asia-east2-c               asia-east2               UP
asia-northeast2-a          asia-northeast2          UP
asia-northeast2-b          asia-northeast2          UP
asia-northeast2-c          asia-northeast2          UP
europe-north1-a            europe-north1            UP
europe-north1-b            europe-north1            UP
europe-north1-c            europe-north1            UP
europe-west6-a             europe-west6             UP
europe-west6-b             europe-west6             UP
europe-west6-c             europe-west6             UP
northamerica-northeast1-a  northamerica-northeast1  UP
northamerica-northeast1-b  northamerica-northeast1  UP
northamerica-northeast1-c  northamerica-northeast1  UP
us-west2-a                 us-west2                 UP
us-west2-b                 us-west2                 UP
us-west2-c                 us-west2                 UP`

**Comment by redborian** [comment]
Thanks for the detailed information @rosera. We will investigate and fix this issue in the upcoming release.
As a workaround, you could set the default project through `gcloud init` command and try creating the cluster again. You can also use our web UI to create the cluster and then use CloudCode to add the cluster to kubeconfig file.

**Comment by redborian** [comment]
We have enhanced the experience in our upcoming release. I will update this thread once we release it.

**Comment by redborian** [comment]
We have fixed this issue in our latest release (v0.0.13)
